server/src/internalClusterTest/java/org/elasticsearch/timeseries/support/TimeSeriesMetricsIT.java

```diff
@@ -80,44 +83,47 @@ public void assertSmallSimple(Object d1, Object d2, CheckedConsumer<XContentBuil
             dimensionMapping.accept(mapping);
             mapping.field("time_series_dimension", true);
             mapping.endObject();
-        });
+            // Add a keyword dimension as a routing parameter
+            mapping.startObject("k").field("type", "keyword").field("time_series_dimension", true).endObject();
```

Comment:
Adding a `keyword` dimension field named `k`, simply because there always must be a routing dimension. It's value is always `k`